tasks/IMPLEMENTATION_ROADMAP.md

```
@@ -0,0 +1,476 @@
+  - [ ] Extract common code from `fix_issue.py`
+  - [ ] Create `coding-agent/lib/github_ops.py`
+  - [ ] Create `coding-agent/lib/job_tracker.py`
+
+### Not Prioritized Yet
+- This phase is explicitly **deferred** until ingestion moat is rock-solid.
+
+---
+
+## Implementation Order Summary
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)